components/Careers/ApplyNewJob.vue

```diff
@@ -0,0 +1,223 @@
+                                              autocomplete="phone"
+                                            />
+                                        </FormField>                                      
+                                    </div>
+                                </div>
+
+                                <div class="lg:col-span-6">
+                                    <div class="ltr:text-left rtl:text-right">
+                                        <FormField label="Job Title:" help="Required. Your Job Title:" :error="errorMsgs.jobTitle">
+                                            <FormControl
+                                              v-model="formData.jobTitle"
+                                              type="email"
+                                              name="email"
+                                              required
+                                              autocomplete="email"
+                                            />
+                                          </FormField>                                      
+                                    </div>
+                                </div>
+
+                                <div class="lg:col-span-6">
+                                    <FormField label="Types of Job:" help="Required. Your Job Type:" :error="errorMsgs.jobType">
+                                        <FormControl
+                                          v-model="formData.jobType"
+                                          type="select"
+                                          name="email"
+                                          required
+                                          autocomplete="email"
+                                          :options="jobTypes"
+                                        />
+                                    </FormField>  
+                                </div>
+
+                                <div class="lg:col-span-12">
+                                    <div class="ltr:text-left rtl:text-right">
+                                        <FormField label="Your Comment" help="Required. Your comment" :error="errorMsgs.comment">
+                                            <FormControl
+                                              v-model="formData.comment"
+                                              type="textarea"
+                                              name="email"
+                                              required
+                                              autocomplete="email"
+                                            />
+                                          </FormField>                                      </div>
+                                </div>
+
+                                <div class="lg:col-span-12">
+                                        <FormField label="Upload file:" help="Max 500kb">
+                                            <FormFilePicker label="Upload" />
```

Comment:
add constraint  of only pdf file under 5MB should be selected

Reply:
i have use accept = ".pdf" this goes to form file picker 
then event of file upload would be handled by that file all changes are to be done there